examples/remote/binary_dependencies/Cargo.toml

```diff
@@ -15,6 +15,7 @@ workspace_path = "//remote/binary_dependencies/cargo"
 gen_workspace_prefix = "remote_binary_dependencies"
 genmode = "Remote"
 default_gen_buildrs = true
+package_aliases_dir = "cargo"
```

Comment:
ugh

Reply:
translation:

what is this field

why is it here

why isn't this some default setting

why is this not mentioned in the PR description.

Reply:
Ultimately doesn't really impact the tests but in the grand scheme of things I do feel that it's more correct to have a named directory and not `"."` for this setting. But that can be discussed later (since this PR is still in draft phase and another PR more directly relates to this)

Reply:
I don't understand how you envision this code review process working...

Reply:
I think the real issue here is that these drafts are stacked on other changes. So while I'm thinking, "oh, that's relevant to another PR", that won't be clear to anyone else.

Reply:
Does this make more sense after reviewing #306 ?

Reply:
Resolving for now